### PR TITLE
plasma-infra: Update private packages version with @auto-it 

### PR DESCRIPTION
**Pull request by Yakutoc**
## Release Notes

Включили обновление версий для всех пакетов, включая приватные. 

### What/why Changed

Сделали патчи для `@auto-it/core` и `@auto-it/npm` что бы можно было включить в обработку все пакеты.

Как оказалось в `@auto-it/core` в методе `getLernaPackages(packages/core/src/utils/get-lerna-packages.ts)` была логика **фильтрующая** пакеты по полю `private`: 

```ts
   if (version !== "MISSING" && isPrivate !== "PRIVATE") {
      packages.push({ path: packagePath, name, version });
    }
``` 

C помощью [`patch-package`](https://github.com/ds300/patch-package) сделал патчи для изменения этой логики.

В этом [commit из песочнице](https://github.com/PoligonSa/plasma-sandbox/commit/4e1c6057e8f242e45f096108293c4d604836fff5) можно посмотреть как это работает. 

Запуск локально проекта `auto next --dry-run` (Report what command will do but do not actually do anything)

при изменениях в пакете `plasma-web`:

```bash
@salutejs/plasma-web@1.228.0-dev.0
@salutejs/plasma-docs-ui@0.139.0-dev.0
@salutejs/plasma-temple-docs@0.164.0-dev.0
@salutejs/plasma-ui-docs@0.213.0-dev.0
@salutejs/plasma-web-docs@0.182.0-dev.0
```

Так же откатили/удалили все что было связанно с ручным обновлением версий в приватных пакетах.  

------
   

<details>
  <summary>:baby_chick: Download canary assets:</summary>
  <blockquote>
    <details>
      <summary>borderRadius</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_default_react-native--canary.734.6245473475.ts'>borderRadius_default_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_b2c_react-native--canary.734.6245473475.ts'>borderRadius_plasma_b2c_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_plasma_web_react-native--canary.734.6245473475.ts'>borderRadius_plasma_web_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberHealth_react-native--canary.734.6245473475.ts'>borderRadius_sberHealth_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_business_react-native--canary.734.6245473475.ts'>borderRadius_sbermarket_business_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_metro_react-native--canary.734.6245473475.ts'>borderRadius_sbermarket_metro_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_react-native--canary.734.6245473475.ts'>borderRadius_sbermarket_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_selgros_react-native--canary.734.6245473475.ts'>borderRadius_sbermarket_selgros_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts'>borderRadius_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberonline_react-native--canary.734.6245473475.ts'>borderRadius_sberonline_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_sberprime_react-native--canary.734.6245473475.ts'>borderRadius_sberprime_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/borderRadius_stylesSalute_react-native--canary.734.6245473475.ts'>borderRadius_stylesSalute_react-native--canary.734.6245473475.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>color</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_ios-swift--canary.734.6245473475.swift'>color_default_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_kotlin--canary.734.6245473475.kt'>color_default_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_react-native--canary.734.6245473475.ts'>color_default_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_default_xml--canary.734.6245473475.xml'>color_default_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_ios-swift--canary.734.6245473475.swift'>color_plasma_b2c_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_kotlin--canary.734.6245473475.kt'>color_plasma_b2c_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_react-native--canary.734.6245473475.ts'>color_plasma_b2c_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_b2c_xml--canary.734.6245473475.xml'>color_plasma_b2c_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_ios-swift--canary.734.6245473475.swift'>color_plasma_web_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_kotlin--canary.734.6245473475.kt'>color_plasma_web_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_react-native--canary.734.6245473475.ts'>color_plasma_web_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_plasma_web_xml--canary.734.6245473475.xml'>color_plasma_web_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_ios-swift--canary.734.6245473475.swift'>color_sberHealth_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_kotlin--canary.734.6245473475.kt'>color_sberHealth_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_react-native--canary.734.6245473475.ts'>color_sberHealth_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberHealth_xml--canary.734.6245473475.xml'>color_sberHealth_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_ios-swift--canary.734.6245473475.swift'>color_sbermarket_business_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_kotlin--canary.734.6245473475.kt'>color_sbermarket_business_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_react-native--canary.734.6245473475.ts'>color_sbermarket_business_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_business_xml--canary.734.6245473475.xml'>color_sbermarket_business_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_ios-swift--canary.734.6245473475.swift'>color_sbermarket_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_kotlin--canary.734.6245473475.kt'>color_sbermarket_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_ios-swift--canary.734.6245473475.swift'>color_sbermarket_metro_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_kotlin--canary.734.6245473475.kt'>color_sbermarket_metro_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_react-native--canary.734.6245473475.ts'>color_sbermarket_metro_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_metro_xml--canary.734.6245473475.xml'>color_sbermarket_metro_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_react-native--canary.734.6245473475.ts'>color_sbermarket_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_ios-swift--canary.734.6245473475.swift'>color_sbermarket_selgros_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_kotlin--canary.734.6245473475.kt'>color_sbermarket_selgros_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_react-native--canary.734.6245473475.ts'>color_sbermarket_selgros_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_selgros_xml--canary.734.6245473475.xml'>color_sbermarket_selgros_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_ios-swift--canary.734.6245473475.swift'>color_sbermarket_wlbusiness_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_kotlin--canary.734.6245473475.kt'>color_sbermarket_wlbusiness_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts'>color_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_wlbusiness_xml--canary.734.6245473475.xml'>color_sbermarket_wlbusiness_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sbermarket_xml--canary.734.6245473475.xml'>color_sbermarket_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_ios-swift--canary.734.6245473475.swift'>color_sberonline_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_kotlin--canary.734.6245473475.kt'>color_sberonline_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_react-native--canary.734.6245473475.ts'>color_sberonline_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberonline_xml--canary.734.6245473475.xml'>color_sberonline_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_ios-swift--canary.734.6245473475.swift'>color_sberprime_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_kotlin--canary.734.6245473475.kt'>color_sberprime_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_react-native--canary.734.6245473475.ts'>color_sberprime_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_sberprime_xml--canary.734.6245473475.xml'>color_sberprime_xml--canary.734.6245473475.xml</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_ios-swift--canary.734.6245473475.swift'>color_stylesSalute_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_kotlin--canary.734.6245473475.kt'>color_stylesSalute_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_react-native--canary.734.6245473475.ts'>color_stylesSalute_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/color_stylesSalute_xml--canary.734.6245473475.xml'>color_stylesSalute_xml--canary.734.6245473475.xml</a><br>
      </blockquote>
    </details>
    <details>
      <summary>shadow</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_default_react-native--canary.734.6245473475.ts'>shadow_default_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_b2c_react-native--canary.734.6245473475.ts'>shadow_plasma_b2c_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_plasma_web_react-native--canary.734.6245473475.ts'>shadow_plasma_web_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberHealth_react-native--canary.734.6245473475.ts'>shadow_sberHealth_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_business_react-native--canary.734.6245473475.ts'>shadow_sbermarket_business_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_metro_react-native--canary.734.6245473475.ts'>shadow_sbermarket_metro_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_react-native--canary.734.6245473475.ts'>shadow_sbermarket_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_selgros_react-native--canary.734.6245473475.ts'>shadow_sbermarket_selgros_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts'>shadow_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberonline_react-native--canary.734.6245473475.ts'>shadow_sberonline_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_sberprime_react-native--canary.734.6245473475.ts'>shadow_sberprime_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/shadow_stylesSalute_react-native--canary.734.6245473475.ts'>shadow_stylesSalute_react-native--canary.734.6245473475.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>spacing</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_default_react-native--canary.734.6245473475.ts'>spacing_default_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_b2c_react-native--canary.734.6245473475.ts'>spacing_plasma_b2c_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_plasma_web_react-native--canary.734.6245473475.ts'>spacing_plasma_web_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberHealth_react-native--canary.734.6245473475.ts'>spacing_sberHealth_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_business_react-native--canary.734.6245473475.ts'>spacing_sbermarket_business_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_metro_react-native--canary.734.6245473475.ts'>spacing_sbermarket_metro_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_react-native--canary.734.6245473475.ts'>spacing_sbermarket_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_selgros_react-native--canary.734.6245473475.ts'>spacing_sbermarket_selgros_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts'>spacing_sbermarket_wlbusiness_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberonline_react-native--canary.734.6245473475.ts'>spacing_sberonline_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_sberprime_react-native--canary.734.6245473475.ts'>spacing_sberprime_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/spacing_stylesSalute_react-native--canary.734.6245473475.ts'>spacing_stylesSalute_react-native--canary.734.6245473475.ts</a><br>
      </blockquote>
    </details>
    <details>
      <summary>typo</summary>
      <blockquote>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_ios-swift--canary.734.6245473475.swift'>typo_mage_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_kotlin--canary.734.6245473475.kt'>typo_mage_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_mage_react-native--canary.734.6245473475.ts'>typo_mage_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_ios-swift--canary.734.6245473475.swift'>typo_plasma_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_kotlin--canary.734.6245473475.kt'>typo_plasma_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_plasma_react-native--canary.734.6245473475.ts'>typo_plasma_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_ios-swift--canary.734.6245473475.swift'>typo_ruler_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_kotlin--canary.734.6245473475.kt'>typo_ruler_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_ruler_react-native--canary.734.6245473475.ts'>typo_ruler_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_ios-swift--canary.734.6245473475.swift'>typo_sage_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_kotlin--canary.734.6245473475.kt'>typo_sage_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sage_react-native--canary.734.6245473475.ts'>typo_sage_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_ios-swift--canary.734.6245473475.swift'>typo_sbermarket_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_kotlin--canary.734.6245473475.kt'>typo_sbermarket_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_sbermarket_react-native--canary.734.6245473475.ts'>typo_sbermarket_react-native--canary.734.6245473475.ts</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_ios-swift--canary.734.6245473475.swift'>typo_soulmate_ios-swift--canary.734.6245473475.swift</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_kotlin--canary.734.6245473475.kt'>typo_soulmate_kotlin--canary.734.6245473475.kt</a><br>
        <a href='https://github.com/salute-developers/plasma/releases/download/v0.0.0-canary/typo_soulmate_react-native--canary.734.6245473475.ts'>typo_soulmate_react-native--canary.734.6245473475.ts</a><br>
      </blockquote>
    </details>
  </blockquote>
</details>

<details>
  <summary>📦 Published PR as canary version: <code>Canary Versions</code></summary>
  <br />

  :sparkles: Test out this PR locally via:
  
  ```bash
  npm install @salutejs/plasma-web@1.228.0-canary.734.6244830667.0
  # or 
  yarn add @salutejs/plasma-web@1.228.0-canary.734.6244830667.0
  ```
</details>

### CODE REVIEW

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/
web storybook: http://plasma.sberdevices.ru/pr/pr-734/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/
web storybook: http://plasma.sberdevices.ru/pr/pr-734/web-storybook/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/
web storybook: http://plasma.sberdevices.ru/pr/pr-734/web-storybook/

**Comment by Salute-Eva** [comment]
**Component performance testing**

Result: **🟢 Success**

Check out report in job artifacts!

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/

**Comment by Salute-Eva** [comment]
Theme Builder app deployed!
  
  http://plasma.sberdevices.ru/pr/plasma-theme-builder-pr-734/

**Comment by Salute-Eva** [comment]
Documentation preview deployed!

  website: http://plasma.sberdevices.ru/pr/pr-734/

File: website/plasma-temple-docs/package.json [hunk 1]
```diff
@@ -30,9 +30,9 @@
         "@docusaurus/theme-live-codeblock": "2.0.0-beta.6",
         "@mdx-js/react": "^1.6.21",
         "@salutejs/plasma-docs-ui": "0.138.0",
-        "@salutejs/plasma-icons": "~1.156.0-dev.0",
-        "@salutejs/plasma-tokens": "~1.60.0-dev.0",
-        "@salutejs/plasma-ui": "~1.203.0-dev.0",
```
Comment (Yeti-or) [inline website/plasma-temple-docs/package.json:35]: а тут будет dev после влития?

**Review by Yeti-or — APPROVED** [review]

File: website/plasma-temple-docs/package.json (hunk 1 above)
Comment (Yakutoc) [reply website/plasma-temple-docs/package.json:35]: @Yeti-or Да, после того как будут выпущены RC версии этих пакетов. 

Сейчас у этих пакетов - latest

**Review by neretin-trike — APPROVED** [review]

**Review by kayman233 — APPROVED** [review]

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Comment by Salute-Eva** [comment]
🚀 This PR is included in version: @salutejs/plasma-temple-docs@0.163.1-dev.0, @salutejs/plasma-ui-docs@0.212.1-dev.0, @salutejs/plasma-web-docs@0.181.1-dev.0 🚀

**Pull request merged**